includes/CLI/Plugin_Check_Command.php

```diff
@@ -522,6 +522,7 @@ private function flatten_file_results( $file_errors, $file_warnings ) {
 				foreach ( $column_errors as $column_error ) {
 
 					$column_error['message'] = str_replace( '<br>', "\n", $column_error['message'] );
+					$column_error['message'] = wp_strip_all_tags( $column_error['message'] );
```

Comment:
Nice!

<details>

<summary>Table(default) format</summary>

FILE: readme.txt

| line | column | type  | code | message | docs |
| ---- | ------ | ----  | ---- | ------- | ---- |
| 0    | 0      | ERROR | outdated_tested_upto_header | Tested up to: 6.5 < 6.6. The "Tested up to" value in your plugin is not set to the current version of WordPress. This means your plugin will not show up in searches, as we require plugins to be compatible and documented as tested up to the most recent version of WordPress. | https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/#readme-header-information |
</details>

<details>

<summary>Json format</summary>

FILE: readme.txt
```
[{"line":0,"column":0,"type":"ERROR","code":"outdated_tested_upto_header","message":"Tested up to: 6.5 < 6.6.\nThe \"Tested up to\" value in your plugin is not set to the current version of WordPress. This means your plugin will not show up in searches, as we require plugins to be compatible and documented as tested up to the most recent version of WordPress.","docs":"https:\/\/developer.wordpress.org\/plugins\/wordpress-org\/how-your-readme-txt-works\/#readme-header-information"}]
```
</details>

<details>

<summary>CSV format</summary>
```
FILE: readme.txt
line,column,type,code,message,docs
0,0,ERROR,outdated_tested_upto_header,"Tested up to: 6.5 < 6.6.
The ""Tested up to"" value in your plugin is not set to the current version of WordPress. This means your plugin will not show up in searches, as we require plugins to be compatible and documented as tested up to the most recent version of WordPress.",https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/#readme-header-information
```
</details>

Not blocker: The JSON format didn't remove `\n` as it will exclude it if we pass second perameter as `true` in `wp_strip_all_tags()`.